contracts/native/version-control/src/commands.rs

```diff
@@ -117,38 +118,44 @@ pub fn approve_or_reject_module(
     ))
 }
 
-/// Remove a module
-pub fn remove_module(
-    deps: DepsMut,
-    msg_info: MessageInfo,
-    module: ModuleInfo,
-    yank: bool,
-) -> VCResult {
-    // if the caller is a namespace owner
-    if !cw_ownable::is_owner(deps.storage, &msg_info.sender)? {
-        // if the module wants to be removed, it must be yanked. Only the admin can remove a module
-        if !yank {
-            return Err(VCError::OnlyYankAllowed);
-        }
-        // validate the caller is the owner of the namespace
-        validate_account_owner(deps.as_ref(), &module.namespace, &msg_info.sender)?;
-    }
+/// Remove a module from the Version Control registry.
+pub fn remove_module(deps: DepsMut, msg_info: MessageInfo, module: ModuleInfo) -> VCResult {
+    // Only the Version Control Admin can remove modules
+    cw_ownable::assert_owner(deps.storage, &msg_info.sender)?;
+
+    // Only specific versions may be removed
+    module.assert_version_variant()?;
+
+    ensure!(
+        MODULE_LIBRARY.has(deps.storage, &module),
+        VCError::ModuleNotFound(module)
+    );
+
+    MODULE_LIBRARY.remove(deps.storage, &module);
```

Comment:
Do we want to be able to remove modules from the yaked modules map as well?

Reply:
Ah I see. Yes, that should be good because this is like a "delete module" action. Should we remove it from pending as well or leave that up to the `rejectModule`?